code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -53,16 +53,13 @@
 	. += span_notice("You can link it with an air sensor using a multitool.")
 
 /obj/machinery/atmospherics/components/unary/vent_pump/multitool_act(mob/living/user, obj/item/multitool/multi_tool)
-	. = ..()
-
```

Comment:
is this necessary?

Reply:
The parent proc just returns true so yeah it's a waste of space